### PR TITLE
[GETTING STARTED] Fixing accessibility issue in checklist

### PR DESCRIPTION
**Pull request by dcaballeroSf**
- Fixing small issue about using the checkbox in the checklist with the keyboard(it used to lose focus after the server call was finished)

# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by dcaballeroSf** [comment]
[W-8461612](https://gus.lightning.force.com/a07B0000008jS6vIAE)

File: src/lwc/gsChecklistItem/gsChecklistItem.js [hunk 1]
```diff
@@ -125,13 +125,11 @@ export default class GsChecklistItem extends NavigationMixin(LightningElement) {
         target.disabled = true;
 
         updateCheckItem({'itemId': this.item.id, 'isChecked': checked})
-        .then (_ => {
+        .then ( _ => {
```
Comment (force2b) [inline src/lwc/gsChecklistItem/gsChecklistItem.js:128]: I had to read up on this syntax (` _ => {}`) as it's not one I've seen before. I'm assuming adding the space before the underscore was not part of the fix in this PR?

Comment (force2b) [reply src/lwc/gsChecklistItem/gsChecklistItem.js:128]: If I'm not mistaken this would also work as `() => {}`, which is how I've generally written these.

**Review by force2b — APPROVED** [review]
I don't really know how to review this or what is being changed?

**Pull request merged**